vnext/Shared/Networking/IRedirectEventSource.idl

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.React.Networking {
```

Comment:
why are we creating a new namespace in the metadata?

Reply:
It's an existing internal namespace, and this interface is not meant to be exported (are all MIDL files added to the DLL metadata and ABI boundary?).

It was created to have a WinRT class to be instantiated as an interface and prevent a circular dependency.
(See `RedirectHttpFilter.cpp` only usage of `OnRedirecting`.

Reply:
it's an existing internal C++-only namespace - there is no such namespace in the metadata afaik; and if you look at the rest of the code in RNW, no other component is following this pattern, only yours :) 
Yes, all WinRT types are public and exposed in the metadata.